index.html

```diff
@@ -0,0 +1,67 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light" id="nav-bar">
+            <a class="navbar-brand" href="#">Sammy Computers</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#home">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#services">Services</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <img src="assets/img/mac.png" alt="" id="header-img">
+
+    </header>
+
+    <video id="video" src="assets/videos/compad.mp4">Compact description video</video>
+
+    <form id="form" method="post" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" id="email" placeholder="Email" name="email">
+        <input type="submit" name="button" value="submit" id="submit">
+    </form>
+
+    <div class="my-sections">
+      <section id="home">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vel, praesentium velit ipsa sint unde adipisci officia sapiente expedita excepturi ducimus pariatur veritatis voluptatum voluptatem consequatur id a aspernatur ex iste.
+      </section>
+
+      <section id="about">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus dolorem molestias earum, consectetur voluptatum magni commodi ducimus soluta consequuntur. Labore officia doloribus suscipit eius numquam culpa consequuntur reprehenderit fugiat atque?
+      </section>
+
+      <section id="services">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt, omnis minus placeat maxime tempore repellat nemo blanditiis, rerum earum unde laudantium magni, incidunt repellendus. Libero, eveniet tempore! Recusandae, aut voluptate.
+      </section>
+    </div>
+
+
+    <script src="js/index.js"></script>
```

Comment:
- [ ] This JavaScript file is missing.

Reply:
I think that you should add a comment which is clear enough and point the issues that are related to that particular file. You can check the comment above about professional communication for more details.